api/v1/openapi.yaml

```diff
@@ -1736,6 +1736,9 @@ definitions:
       host-firewall:
         description: Status of the host firewall
         "$ref": "#/definitions/HostFirewall"
+      Srv6:
```

Comment:
This should be lower case to be consistent with other entries on the same level.